readme.md

```diff
@@ -1,72 +1,32 @@
-
-#### Option B - ....via 'Full Zip Download' ####
-Download the complete GIT repo and extract it into the Kodi Add-on folder.
+There are two ways to install Retrospect, depending on what version of Kodi you are using:
 
-#### Caution!
-Be aware that if you install it using *Option A* or *Option B* Kodi will **disable** the add-on by default and it will **not install any of the dependencies**. So you need to enable it manually and install all the dependencies by hand. Starting from Kodi Leia this is easier as it has a "View Dependencies" option of add-ons.   
+### Kodi Leia and later
+Starting from Kodi Leia (v18), you can easy install Retrospect via the Kodi repository. Either search the add-ons in the Kodi repository for Retrospect and install it. More detailed information can be found in the [Retrospect Wiki](https://github.com/retrospect-addon/plugin.video.retrospect/wiki/Installation).
 
+**Note:** Make sure that the _Auto-Update_ option for Retrospect is enabled to automagically receive new updates.
 
-### 3 - Updating Retrospect ###
+![alt text](./plugin.video.retrospect/resources/media/retrospect01.jpg "The Retrospect information screen")
 
-#### Option A - ....via 'GIT Clone' ####
-Pull latest changes into your clone (located in the Kodi Add-on folder). After that remove all existing `*.pyc` and `*.pyo` files within the Retrospect folders (**don't skip this**).
+### Kodi Krypton    
+If you are running Kodi Krypton (v17) the only way to install Retrospect is to install it from a zip file. Follow these steps (keep in mind they might slightly differ depending on your skin):
 
-#### Option B - ....via 'Full Zip Download' ####
-Download the complete GIT repo. Remove all existing Retrospect folders (**don't skip this**) and extract the new ones it into the Kodi Add-on folder.
+1. Download the latest Retrospect zip file from here: [https://github.com/retrospect-addon/plugin.video.retrospect/releases](https://github.com/retrospect-addon/plugin.video.retrospect/releases)
+1. Transfer the file to you Kodi system (_optional_)
+1. Install Retrospect from zip file
 
-#### Finalizing the update ####
-In both situation run Retrospect at least once before accessing the Retrospect add-on settings. The initial run might take longer than usual, as Retrospect is initialising some stuff and downloads artwork.
+More detailed information can be found in the [Retrospect Wiki](https://github.com/retrospect-addon/plugin.video.retrospect/wiki/Installation).
 
-# ! Be advised ! #
-Retrospect will NOT auto-update. So new version need to be installed manually. 
+**Note:** Since Retrospect isn't available in the official Kodi add-on repository for Kodi 17 Krypton, it will not automagically update. Retrospect will notifiy you of new versions. You will then have to manually update you Retrospect add-on using the steps above.
 
 # Troubleshooting #
```

Comment:
I would move the complete Troubleshooting section to the Wiki and only refer to it briefly.
The README should simply be the basics for a beginning user.
- What is it? (to determine if it is useful to me)
- How do I get it? (make it simple, otherwise they may not even bother)
- Where do I get more information? (in case the above doesn't suffice/work)

The more information you put in the README, the less likely it is that users read it fully (or even bother to read that what *is* important). So stick to the minimum, and offer a solution if that is not sufficient.

Reply:
Besides, Widevine installation should work out-of-the-box, and if it doesn't, users should go to the InputStream Helper documentation. So I'd rather have a good set of documentation centralized in that project, rather than scattered over the Internet for various add-ons (possibly outdated instructions)...